pkg/backup/backup_test.go

```diff
@@ -718,7 +718,6 @@ func TestBackupAndRestoreJobDescription(t *testing.T) {
 	}
 
 	sqlDB.Exec(t, "BACKUP INTO ($1, $2, $3)", collections...)
-	sqlDB.Exec(t, "BACKUP INTO LATEST IN ($1, $2, $3)", collections...)
```

Comment:
Why was this removed from the test?

Reply:
I removed it mainly for simplicity's sake. The test just checks the description outputted once the backup statement is fully resolved, so in this case, `INTO LATEST` just checks that `LATEST` is resolved accordingly.

We do already have a `BACKUP INTO LATEST ... WITH incremental_location` case which covers `LATEST`. If we wanted to still have them tested separately, it just would require a decent amount of changes to the test since you'd have to also update the logic for finding the full path that `LATEST` gets resolved into and separate the two full paths.